.travis.linux.trusty.sh

```diff
@@ -0,0 +1,21 @@
+#! /bin/bash
+
+URL=ftp://ftp.simplesystems.org/pub/libpng/png/src/libpng16/libpng-1.6.23.tar.gz
+FILE=libpng-1.6.23.tar.gz
```

Comment:
You'll probably want to also set `PNG_STATIC` before adding the subdirectory (and disable `PNG_SHARED` and `PNG_TESTS`) to statically link it.